lib/schedule.js

```diff
@@ -327,7 +341,7 @@ Range.prototype.contains = function(val) {
 
   NOTE: Cron months are 1-based, but RecurrenceRule months are 0-based.
 */
-function RecurrenceRule(year, month, date, dayOfWeek, hour, minute, second) {
+function RecurrenceRule(year, month, date, dayOfWeek, hour, minute, second, timezone) {
   this.recurs = true;
```

Comment:
I'm not too sure I would attach the `timezone` to the `RecurrenceRule` but to the `Job` object. I think it makes more sense as in the API we're separating the `spec` from the `timezone` option. As an example of what I'm trying to say see [this](https://github.com/node-schedule/node-schedule/commit/ce8d7a6b27c0ead77b15d031de23a28be44cb096) (once upon a time I tried to implement the timezone support but failed because I was using `node-time` that wasn't really portable)